include/materials/mohr_coulomb.tcc

```diff
@@ -399,7 +311,7 @@ void mpm::MohrCoulomb<Tdim>::compute_df_dp(
             ((sin(phi) * sin(theta + M_PI / 3.) /
               (std::sqrt(3.) * cos(phi) * cos(phi))) +
              (cos(theta + M_PI / 3.) / (3. * cos(phi) * cos(phi)))) +
-        (mean_p / (cos(phi) * cos(phi)));
+        (mpm::materials::p(stress) / (cos(phi) * cos(phi)));
```

Comment:
The `p` function in `mpm::materials` give you tension positive. Just making sure, your `p` definition here also tension positive right?

Reply:
Yes, I have passed all test, and they should be correct.

Reply:
I see, good then!